frontend/store/auctions.ts

```diff
@@ -52,6 +55,9 @@ export const getters = {
     getAuctionById: (state: State) => (id: string) => {
         return state.auctionStorage[id];
     },
+    getEventsByID: (state: State) => (id: string) => {
+        return state.eventStorage[id];
+    },
```

Comment:
Let's call it `takeEventsStorage` since we may have more types of events in the future or account for them here:
```suggestion
    getTakeEventsByID: (state: State) => (id: string) => {
        return state.eventStorage[id];
    },
```